project/frontend/src/FindMatch/FindMatch.js

```diff
@@ -0,0 +1,45 @@
+import React from "react"
+import ReactDOM from "react-dom"
+
+class PreferenceForm extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { cuisine: "" }
+    this.changeState = this.changeState.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+
+  // Updates the state of the input element so it holds the chosen value.
+  changeState(event) {
+    this.setState({ cuisine: event.target.value })
+  }
+
+  handleSubmit(event) {
+    alert("the cuisine you chose is: " + this.state.cuisine)
+  }
```

Comment:
Let's also add the event.preventDefault() at the end here so our page doesn't refresh.